packages/allure-playwright/test/fixtures.ts

```diff
@@ -87,13 +87,18 @@ const runPlaywrightTest = async (
   if (additionalArgs) {
     args.push(...additionalArgs);
   }
-  const testProcess = fork(require.resolve("@playwright/test/cli"), args, {
+
+  const modulePath = require.resolve("@playwright/test/lib/cli");
+  await base.step(`${modulePath} ${args.join(" ")}`, () => {});
```

Comment:
why we doing this?

Reply:
To display executed commands in the report:

<img width="766" alt="Screenshot 2023-05-26 at 10 43 36" src="https://github.com/allure-framework/allure-js/assets/2149631/dbc4a72c-a10f-4331-9bfa-5d9ad83774c1">